app/controllers/participants_controller.rb

```diff
@@ -4,6 +4,11 @@ class ParticipantsController < ApplicationController
 
   # GET /participants
   # GET /participants.json
+<<<<<<< HEAD
+  # def index
```

Comment:
merge conflict here, please keep this:
`!logged_in?
      require_user`
it's a useful feature, it's also tested